portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,37 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function loadContent() {
+    // addName();
```

Comment:
Remove these lines.

Reply:
- Seems that each div cannot add its own `onload` attribute? https://stackoverflow.com/questions/4057236/how-to-add-onload-event-to-a-div-element
- As `onload` in `<body>` only supports 1 function, I decided to use the `loadContent()` function to call multiple functions. Probably there exists a better way...

Reply:
Currently just add `onload` in `<Body>` SGTM.